lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -102,6 +111,7 @@ def get_virtual_facts(self):
             if not found_virt:
                 virtual_facts['virtualization_type'] = systemd_container
                 virtual_facts['virtualization_role'] = 'guest'
+                virtual_facts['container'] = True
                 found_virt = True
 
```

Comment:
i would probably rewrite this to 
```
virtual_facts['container'] = bool(virtual_facts['virtualization_role'] == 'guest' and virtual_facts['virtualization_type'] in ('podman', 'container', 'docker', ....)
```
inserted right before we return values.

Reply:
I like it! Much cleaner and probably (hopefully) will be easier to maintain. I'll update the PR to reflect the new changes